### PR TITLE
Updated About section in Tutorial.rst

### PR DESCRIPTION
**Pull request by mike-ferguson**
Changed about description to be more precise

### CODE REVIEW

**Review by mschrimpf — APPROVED** [review]

**Comment by mschrimpf** [comment]
I'm not sure why this unit test fails:
```

/om2/group/dicarlo/jenkins/miniconda3/envs/brain-score_unittests_469/lib/python3.7/site-packages/model_tools/brain_transformation/temporal.py:6: in <module>
    from brainio.assemblies import merge_data_arrays, walk_coords
E   ModuleNotFoundError: No module named 'brainio'
```

File: docs/source/modules/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+you most likely will only need a single repo to get a model up and scored. It is
+good to see, however, the structure of how Brain-Score operates with its
+various components. They are:
+
+1. ``brain-score``: the heart of the code that runs analysis and comparisons.
+2. ``sample-model-submission``: template and examples for model submissions.
+3. ``candidate_models``: various pre-trained models/models that have already been scored.
+4. ``model-tools``: helper functions that translate from machine learning models
+   to brain models to be tested on brain-score.
+5. ``brainio_collection``: the repo that packages and collects the stimuli/data.
```
Comment (jjpr-mit) [inline docs/source/modules/tutorial.rst:98]: Has references to `brainio_collection` and `brainio_base`.  But I'm planning to take care of that in another PR.

**Comment by jjpr-mit** [comment]
> I'm not sure why this unit test fails (see jenkins log):

I see why the tests are failing.  [This fork](https://github.com/mike-ferguson/brain-score/blob/b2bbad76b8e7ffa75d746729d21902878470ecda/setup.py) is descended from an old version of master that used brainio_base and brainio_collection.  Lots of changes have happened since then.  @mike-ferguson I think you'll have to merge the current master into your fork.

File: docs/source/modules/tutorial.rst (hunk 1 above)
Comment (mike-ferguson) [reply docs/source/modules/tutorial.rst]: I resolved the conflicts, so we should be good :)

**Review by jjpr-mit — APPROVED** [review]

**Pull request merged**